client/components/close-on-escape/test/index.jsx

```diff
@@ -0,0 +1,75 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+import React from 'react';
+import ReactDom, { unmountComponentAtNode } from 'react-dom';
+import { spy } from 'sinon';
+
+/**
+ * Internal dependencies
+ */
+import useFakeDom from 'test/helpers/use-fake-dom';
+import CloseOnEscape from '../';
+
+const simulateEscapeKeydown = () => (
```

Comment:
I would say normally if we're testing key events we would use:
```
import { mount } from 'enzyme';
const wrapper = mount( <ComponentWithKeyHandler /> );
const wrapper.find( 'selector-to-element-with-handler' ); 
node.simulate( 'keyDown', {
	keyCode: 27
} );
```

In this case though the listener is on the document. @gziolo any thoughts here?